### ISSUE TITLE
Modele etter OpenStreetMap

### ISSUE THREAD

**Issue by darthlukan** [asbjornenge/refugeeroute#1]
Kanskje det vil bli godt å bruke modelen av OpenStreetMap hvor man kan sende data og det blir oppdatert "live."

Også, OpenStreetMap har data som er mer oppdatert enn Google Maps, så hvis du bruke det så kan du har nær "Real-time" data.

**Comment by asbjornenge** [comment]
Jepp, OpenStreetMap er helt klart et alternativ til kartløsning :+1: Spm. er om vi ønsker "community contributions" til løsningens kart? Hva med abuse? Maybe there are a lot of jerks want to mess with the map?

**Comment by darthlukan** [comment]
Ja, abuse er en mulighet, men hvis vi tenker på aliertene som "authed contributors" så kan vi bli litt mer sikkert abuse er kontrollert.

**Comment by darthlukan** [comment]
Også, vi kan gjøre sammenligninger mellom bilder sendt via skjema og OpenStreetMap bildene for å verifisere lokasjonene er "legit". Kreve et bilde med GPS data for å bidra til kartet.

**Comment by asbjornenge** [comment]
@darthlukan true :+1: Mange muligheter og mange som vil bidra også. Vi kunne hatt contributer accounts etc. Men litt down the road foreløpig.

**Comment by asbjornenge** [comment]
Oh, it just hit me. We can make it possible for anyone "on the road" (refugees themselves) to upload positions, photos, videos and stories from their own phones into refugeeroute? Perhaps this was your original idea? :stuck_out_tongue_closed_eyes: Anyway, I get it now and that would be really powerful! And my "headcam" idea might not be necessary since many refugees have smartphones!

**Comment by darthlukan** [comment]
Ja, det var min mening :)

**Comment by asbjornenge** [comment]
:smile: :tada:

**Comment by asbjornenge** [comment]
@darthlukan OpenStreetMaps seems like a great base-map. But it's only data. They don't have a js map lib I can use to build my app with...? I might be wrong. I had a look at mapbox (https://www.mapbox.com/) which seems more what we need. It uses OpenStreetMaps for road data. We would add additional data that might not belong in OpenStreetMaps, or would they? I don't know what types of data you can add there.

**Comment by asbjornenge** [comment]
Other alternatives:
- http://openlayers.org/
- http://leafletjs.com/
- https://www.mapbox.com/

**Comment by darthlukan** [comment]
@asbjornenge - It's built modularly, so you use OpenStreetMap data with whatever mapping GUI you want. The most important part (the data) is already taken care of but you're not locked into a particular UI lib.

**Comment by asbjornenge** [comment]
:+1:

**Comment by darthlukan** [comment]
A data scientist friend of mine suggested https://www.ushahidi.com/ as a possible framework that would work well for this concept via Google+. Something to think about.
